docs/en/04_Changelogs/4.9.0.md

```diff
@@ -1,6 +1,40 @@
 # 4.9.0 (Unreleased)
 
+## Overview
 
-## New features
+- [Features and enhancements](#features-and-enhancements)
+    - [Image lazy loading](#image-lazy-loading)
+    - [Other new features](#other-features)
+- [Bugfixes](#bugfixes)
+
+
+## Features and enhancements {#features-and-enhancements}
+
+### Image lazy loading {#image-lazy-loading}
+
+Most images will now get the loading="lazy" attribute added to them.  This includes images added via the WYSIWYG insert 
+media button as well as image DataObjects added to templates.  Image HTML tags (`&lt;img&gt;`) added in templates will not
+be lazy loaded by default
+
+To disable lazy loading for an individual image in a template use `$MyImage.LazyLoad(false)`.
+
+To out of lazy loading globally, notably if you already have a custom lazy loading implementation, use the following yml config:
+
+```yml
+SilverStripe\Assets\Image:
+  lazy_loading_enabled: false
+```
+
+CMS authors can disable lazy loading on images added via the WYSIWYG editor in the "Edit image" panel by setting the new 
+"Loading" field to "Eager".
+
+Developers who have customised the "Edit image" panel should note the `Editor.Placement.Dimensions` field has been moved to
```

Comment:
We should have a dedicated "How to upgrade" section that lists all the gotchas and link off to the relevant doc page.

Reply:
> to the relevant doc page.

I think that's this page :-)

Have added an 'Upgrade notes' heading

Reply:
Have refactored out the nested FormGroup, so have removed this bit